pkg/operator/ceph/cluster/mgr/spec_test.go

```diff
@@ -70,7 +72,7 @@ func TestPodSpec(t *testing.T) {
 
 	podTemplate := cephtest.NewPodTemplateSpecTester(t, &d.Spec.Template)
 	podTemplate.RunFullSuite(config.MgrType, "a", appName, "ns", "ceph/ceph:myceph",
-		"100", "1337" /* resources */)
```

Comment:
For testing, it seems like these values are the same each time. Could we assume for now that they are the same and not add extra arguments to the function call unless we need to later?

Reply:
Did you intend to change this and miss the update?

Reply:
Thing is tests won't pass if I don't do this, so I kept it this way.